hashes/src/lib.rs

```diff
@@ -24,10 +24,14 @@
 //! # #[cfg(feature = "std")] {
 //! use bitcoin_hashes::Sha256;
 //!
+//! # fn hash() -> Result<Sha256, std::io::Error> {
 //! let mut reader: &[u8] = b"hello"; // In real code, this could be a `File` or `TcpStream`.
 //! let mut engine = Sha256::engine();
-//! std::io::copy(&mut reader, &mut engine).unwrap();
```

Comment:
Maybe this one should be `.expect("engine writes don't error");`

Reply:
done

Reply:
Is it not possible to use `?` in these examples?